crates/holochain_conductor_api/src/app_interface.rs

```diff
@@ -207,6 +216,37 @@ impl From<InstalledAppInfoStatus> for AppStatus {
     }
 }
 
+#[derive(Clone, Debug, PartialEq, Eq, serde::Serialize, serde::Deserialize, SerializedBytes)]
+pub struct DnaGossipInfo {
+    pub total_historical_gossip_throughput: HistoricalGossipThroughput,
+}
+
+/// Throughput info specific to historical rounds
+#[derive(
+    Clone, Debug, Default, PartialEq, Eq, serde::Serialize, serde::Deserialize, SerializedBytes,
+)]
+pub struct HistoricalGossipThroughput {
+    /// Total number of bytes expected to be sent for region data (historical only)
+    pub expected_op_bytes: InOut,
+    /// Total number of ops expected to be sent for region data (historical only)
+    pub expected_op_count: InOut,
+    /// Total number of ops sent
+    pub op_count: InOut,
+    /// Total number of bytes sent for op data
+    pub op_bytes: InOut,
```

Comment:
Just sorting and spacing.

```suggestion
    /// Total number of bytes expected to be sent for region data (historical only)
    pub expected_op_bytes: InOut,

    /// Total number of ops expected to be sent for region data (historical only)
    pub expected_op_count: InOut,

    /// Total number of bytes sent for op data
    pub op_bytes: InOut,

    /// Total number of ops sent
    pub op_count: InOut,
```